tasks/src/tabular-regression/about.md

```diff
@@ -3,7 +3,18 @@
 Tabular regression is the task of predicting a numerical value given a set of attributes/features. *Tabular* meaning that data is stored in a table (like an excel sheet), and each sample is contained in its own row. The features used to predict our target can be both numerical and categorical. However, including categorical features often requires additional preprocessing/feature engineering (a few models do accept categorical features directly, like [CatBoost](https://catboost.ai/)). An example of tabular regression would be predicting the weight of a fish given its' species and length.
 
 ## Use Cases
+Tabular data is one of the most prevalent data out there, and many raw-datasets once we extract features from them can be arrange in tabular form. Below you can find some of the popular tabular use-cases:-
 
+- **Preditcting a continuous target value**: 
+  When the data has been shared to us as a table, say in a spreadsheet in this case we can use the set of attributes/features to predict our target. Some popular examples like, predicting Taxi Fare based on distance traveled and time taken, predicting the Price of House based on number of rooms, square ft. area,  etc.
```

Comment:
Can you avoid using passive voice and simplify the language 🙂

Reply:
Sure I'll change it. Thanks a lot for all the helpful feedback!